Header header
geometry_msgs/Point tip_setpoint
uint32 step_count